consensus/blocknode.go

```diff
@@ -58,143 +41,103 @@ type blockNode struct {
 	delayedSiacoinOutputs map[SiacoinOutputID]SiacoinOutput
 }
 
-// CheckTarget returns true if the block id is lower than the target.
-func (b Block) CheckTarget(target Target) bool {
-	blockHash := b.ID()
-	return bytes.Compare(target[:], blockHash[:]) >= 0
```

Comment:
this is more of a meta-comment, but I wish github made it easier to see where a function was moved to. Pretty inconvenient to have to search through 2k of diffs to learn that it moved to types.go

types.go seems like a good place to put it.